selenium-tests/login.test.js

```diff
@@ -0,0 +1,50 @@
+import {Builder, By, until} from "selenium-webdriver";
+import {getDriver, getElementByXPath, sleep, urls} from "./utils";
+
+const firefox = require("selenium-webdriver/firefox");
+
+describe("Selenium tests for <Login />", () => {
+  let driver;
+  beforeAll(async () => {
+    driver = await getDriver(Builder, new firefox.Options().headless());
+  });
+
+  afterAll(async () => driver.quit());
+
+  it("should render login page and submit login form", async () => {
+    await driver.get(urls.login);
+    const username = await getElementByXPath(
+      driver,
+      "//INPUT[@id='username']",
+      until,
+      By,
+    );
+    const password = await getElementByXPath(
+      driver,
+      "//INPUT[@id='password']",
+      until,
+      By,
+    );
+    username.sendKeys("openwisp@openwisp.org");
+    password.sendKeys("test_password");
+    const submitBtn = await getElementByXPath(
+      driver,
+      "//INPUT[@type='submit']",
+      until,
+      By,
+    );
+    await sleep(1000);
```

Comment:
There should be a better way to do this. 
Last time I used selenium, I used a method that allowed to wait until a specific element on the page was visible/present, with a timeout. 
See FluentWait in this page: https://www.selenium.dev/documentation/en/webdriver/waits/ for more info.
Applies to all other occurrences of `sleep`.